pom.xml

```diff
@@ -511,11 +511,47 @@ under the License.
                 <pomExclude>pom.xml</pomExclude>
               </pomExcludes>
               <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
+              <filterProperties>
+                <repository.proxy.url>${repository.proxy.url}</repository.proxy.url>
+              </filterProperties>
+              <addTestClassPath>true</addTestClassPath>
+
               <goals>
                 <goal>site</goal>
               </goals>
             </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>mrm-maven-plugin</artifactId>
+            <version>1.6.0</version>
+            <configuration>
+              <propertyName>repository.proxy.url</propertyName>
+              <repositories>
+                <mockRepo>
+                  <source>src/it/mrm/repository</source>
+                  <!--
```

Comment:
Let's keep the comment for now, but will be incorrect as soon as other ITs will start to use it.

Reply:
I tried to set the .jar file in the root of the repo, but what is actually served by the mock repo is an empty .jar file.
Alternatively, I could try not using the mrm-maven-plugin, and instead use a setup.groovy preBuildHookScript to copy the actual jars to the IT /local-repo.

Reply:
Using the plugin is totally fine, I do not object. I was solely referring that the comment bears the JIRA issue id. If more ITs will use it, the comment won't be accurate anymore.